packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.js

```diff
@@ -56,27 +61,32 @@ class TranscribedLines extends React.Component {
         previousAnnotationValuesForEachMark.push(previousAnnotationValuesForThisMark)
       })
       mark.setPreviousAnnotations(previousAnnotationValuesForEachMark)
-      mark.finish()
+      if (mark.finished) {
+        const markBounds = node?.getBoundingClientRect()
+        mark.setSubTaskVisibility(true, markBounds)
```

Comment:
This creates a UX where the subtask still opens and it can look like you can create a new mark again for a previous annotation that already has one. It doesn't, but it's confusing. I'd like to however get this in place and walk through these changes and get more feedback from @snblickhan this week.

Reply:
Yeah, I think this is very confusing too but having a line/button that's clickable yet does nothing is also confusing. If the magenta line does nothing, once a green line exists, then it should be disabled (or maybe removed completely so that there's only ever one line per annotation.)

I also find it a bit confusing that clicking a magenta line opens the popup with previous annotations attached to the mark, but clicking the green line opens the popup with the regular text editor. I guess that's so you can change the line that you previously chose? If you made a mistake, and want to choose a different line of text, you'd be expected to delete the green line and start again from the magenta line?